migrations/index.js

```diff
@@ -0,0 +1,36 @@
+const Umzug = require('umzug');
+const config = require('./../config/');
+const { sequelize } = require('../app/models');
+const logger = require('../app/logger');
+
+exports.check = () => {
```

Comment:
This handle the database migrations, loading the files in the directory and comparing if there is a need of running the migration or not, against the database.